### PR TITLE
add retry on JSONDecodeError. fix issue #105

### PR DESCRIPTION
**Pull request by gs202**
fix issue #105
https://github.com/rhevm-qe-automation/pytest_jira/issues/105

### CODE REVIEW

**Comment by gs202** [comment]
JSONDecodeError class is available starting from 3.5.x.
I can set the retry on ValueError to support 3.4.x

**Comment by lukas-bednar** [comment]
Since python 3.4 reached end of live on March of 2019, we can drop 3.4.x matrix.

**Comment by gs202** [comment]
So can it be merged?

**Comment by lukas-bednar** [comment]
First we need to make it green.
Please remove py3.4 from matrix:
https://github.com/rhevm-qe-automation/pytest_jira/blob/master/tox.ini#L4
https://github.com/rhevm-qe-automation/pytest_jira/blob/master/setup.cfg#L22
https://github.com/rhevm-qe-automation/pytest_jira/blob/master/.travis.yml#L3

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,6 @@
 [tox]
 envlist = py34-pytest2,py34-pytest3,py34,py35,py36,py37,pep8
```
Comment (lukas-bednar) [inline tox.ini]: Could you please remove also `py34-pytest2,py34-pytest3,py34` from here ?

In addition, we can get rid of lines which are describing these targets down bellow:
```
[testenv:py34-pytest2]
...
[testenv:py34-pytest3]
...
```

And then we can remove also files  `requirements-pytest2.txt` & `requirements-pytest3.txt` !

Comment (gs202) [reply tox.ini]: done

**Comment by lukas-bednar** [comment]
Thank you.

**Pull request merged**